Resources/Prototypes/Catalog/Fills/Lockers/misc.yml

```
@@ -25,6 +25,12 @@
       children:
       - id: EmergencyOxygenTankFilled
       - id: OxygenTankFilled
+    # Carpmosia-start - Nitrogen Tanks for nitrogen-breathing species.
+    - !type:GroupSelector
+      children:
+      - id: EmergencyNitrogenTankFilled
+      - id: NitrogenTankFilled
+    # Carpmosia-end - Nitrogen Tanks for nitrogen-breathing species.
```

There exists a dedicated N2 emergency locker, with it's own table. They should be meaningfully different.
```suggestion
    # Carpmosia-start - Accessible nitrogen
        weight: 3
    - id: EmergencyOxygenTankFilled
      prob: 0.4
    - id: EmergencyNitrogenTankFilled
      prob: 0.2
    # Carpmosia-end - Accessible nitrogen
```

---

Maybe just keep normal lockers as is? We added nitrogen tanks everywhere else, including fire lockers